### PR TITLE
Merging G&R and Maker Relay

### PR DESCRIPTION
**Pull request by twblack88**
- Ep 105 of Governance and Risk
- maker relay 7
- maker relay 8

### CODE REVIEW

File: governance/communications/maker-realy-ep-07.md [hunk 1]
```diff
@@ -0,0 +1,267 @@
+# Maker Relay 07
```
Comment (LongForWisdom) [inline governance/communications/maker-realy-ep-07.md:1]: Filename is spelt incorrectly. maker-realy-ep-07.md  -> maker-relay-ep-07.md

**Comment by twblack88** [comment]
@LongForWisdom fixed summary and typo in filename for relay

File: governance/communications/maker-realy-ep-07.md (hunk 1 above)
Comment (twblack88) [reply governance/communications/maker-realy-ep-07.md:1]: looks like 7 is missing. Episode zero is the one we didn't deploy on time to the forums or reddit. A dry-run relay. 

Screen Shot 2020-08-24 at 12 12 50 PM

Comment (twblack88) [reply governance/communications/maker-realy-ep-07.md]: i deleted it on the branch. Hopefully that fixes it. 🙏

**Comment by twblack88** [comment]
I think that catches all the missing `MakerDAO's` in the relays and fixes the weird "renamed but not deleted" file that is ep 7.

**Review by amy-jung — APPROVED** [review]
Looks good

**Comment by Davidutro** [comment]
Just a heads up, I submitted a PR with 106, [here](https://github.com/makerdao/community/pull/749)

It is based on the most recent tb-gov branch(that this PR is coming from) so I don't think anything should break.

**Review by LongForWisdom — APPROVED** [review]

**Pull request merged**